docs/reference/deploy/master-config-reference.rst

```diff
@@ -689,7 +703,7 @@ documentation on :ref:`resource-pools` for more information. Defaults to a resou
 ``pool_name``
 =============
 
-The name of the resource pool.
+The name of the resource pool, which must be unique among all resource pools defined.
```

Comment:
Specifies the name of the resource pool, which must be unique among all defined resource pools.